backend/src/routes/payments.js

```diff
@@ -0,0 +1,58 @@
+const express = require('express');
+const services = require('../services');
+const router = express.Router();
+const db = require('../db');
+const auth = require('../auth');
+const utils = require('../utils');
+
+const getPaymentRedirectUrl = async (req, res) => {
+    try {
+        const paymentObj = req.body;
+        const validationErrors = utils.payments.validateAmount(
+            paymentObj.amount
+        );
+        if (validationErrors) {
+            res.status(422).json(validationErrors);
+        } else {
```

Comment:
Please return after the previous line so that the main logic doesn't need to be in its own else block but can exist in the main level of function. This makes functions easier to read as you don't need to check which block you are in but instead you just have the validation logic in the beginning and then the main logic after that.